Code/Framework/AzToolsFramework/AzToolsFramework/UI/DocumentPropertyEditor/DocumentPropertyEditor.h

```diff
@@ -44,12 +69,13 @@ namespace AzToolsFramework
 
     protected:
         DocumentPropertyEditor* GetDPE();
+        DPERowWidget* GetLastDescendentInLayout();
```

Comment:
uber-nit: descendant

Reply:
spelling mistakes aren't nits, they're human compile errors! Fixed in multiple places.